modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/AbstractPageMemoryTableStorage.java

```diff
@@ -93,6 +93,9 @@ public StorageIndexDescriptorSupplier getIndexDescriptorSupplier() {
      */
     public abstract DataRegion<?> dataRegion();
 
+    /** Returns storage engine for the table. */
+    public abstract AbstractPageMemoryStorageEngine engine();
```

Comment:
`protected` maybe? Why would it be public.
Whatever you decide

Reply:
This method is used for example in package `org.apache.ignite.internal.storage.pagememory.mv`.